lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -31,6 +31,35 @@ module Resources
           },
           'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/net-price'
         },
+        {
+          'extension': [
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'http://hl7.org/fhir/sid/ndc',
```

Comment:
I think we have to provide a terminology binding for the relevant fields, for both standard and proprietary codes.
Applicable for the other fields as well, whichever is relevant.

Reply:
Yes. I also think we need them for currencies (https://hl7.org/fhir/r4/valueset-currencies.html) for the amount extensions

Reply:
For Currencies the terminology binding has already been defined ill attach the screenshots.
Unit Price: 
![image](https://user-images.githubusercontent.com/34351581/132453190-2fe04b30-4838-4e86-82c5-7267b7761732.png)

Net Price: 
![image](https://user-images.githubusercontent.com/34351581/132453233-1ccea0c7-d681-4477-bd6d-5d16083d2dfc.png)

Reply:
I do not think this entries belong to the terminology bindings section. I think it probably comes from the extensions section.
We would need to add to the section for terminology bindings here:
https://fhir.cerner.com/millennium/r4/financial/general/charge-item/#terminology-bindings

Reply:
Yes. We have added Currencies to the Terminology bindings for other valueMoney extensions like with FinancialTransaction http://fhir.cerner.com/millennium/r4/financial/general/financial-transaction/#terminology-bindings and Account http://fhir.cerner.com/millennium/r4/financial/general/account/#terminology-bindings

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/727/commits/1b1988a56f859d4c0a35e2d9be07fff02566badc#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR109-R134

Reply:
Thank you for making the change. Can we format <code>net-price</code> and <code>unit-price</code> to match the other extensions in the terminology bindings. I also think we should change the unrelated terminology bindings above that have <extension></extension> to match.

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/727/commits/283cb2769c467a5e66ae29c59b12e0f3e2cc834c#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR116